svc-systems/plugin/client.go

```diff
@@ -23,6 +23,7 @@ import (
 	"io/ioutil"
 	"log"
```

Comment:
why is this using go lang log ? Shouldnt it use logrus ??

Reply:
Still not resolved ?

Reply:
this is addressed